doc/README_CN.md

```diff
@@ -16,59 +16,62 @@
 (_______)_______/  )_(  |/     \|   v0.10
 ```
 
-# Git 啊：管理多个 git 库的命令行工具
+# Gita：一个管理多个 git 库的命令行工具
 
-老夫有两把刷子：
+这个工具有两个作用：
 
-- 同时显示多个库的状态信息，比如库杈名，编辑状态，贡献信息等
+- 并排显示多个库的状态信息，比如分支名，编辑状态，提交信息等
 - 在任何目录下代理执行 git 指令
 
 ![gita screenshot](https://github.com/nosarthur/gita/raw/master/doc/screenshot.png)
 
-库杈的五色代表了本土和远程库杈相生相杀的关系：
+本地和远程分支之间的关系有5种情况，在这里分别用5种颜色对应着：
 
-- 青：远近一致
-- 赤：本土已与远程分道扬镳
-- 黄：本土落后于远程（宜和合merge）
-- 白：没有远程
-- 紫：本土超前于远程（宜推送push）
+- 绿色：本地和远程保持一致
+- 红色：本地和远程产生了分叉
+- 黄色：本地落后于远程（适合合并）
+- 白色：本地没有指定远程
+- 紫色：本地超前于远程（适合推送）
 
-如果你知道蓝移和红移的道理，就会明白为什么黄色表示落后，紫色表示超前。
+为什么选择了紫色作为超前以及黄色作为落后，绿色作为基准 的理由在这两篇文章中解释：  
+[blueshift](https://en.wikipedia.org/wiki/Blueshift)、[redshift](https://en.wikipedia.org/wiki/Redshift)
 
-编辑状态符号：
+额外的状态符号意义：
 
-- `+`: 已上台(staged)
-- `*`：未上台(unstaged)
-- `_`：未追踪(untracked)
+- `+`: 暂存(staged)
+- `*`： 未暂存（unstaged）
+- `_`： 未追踪（untracked）
 
-基础指令：
+记录的子命令：
```

Comment:
I prefer the original translation for this line

Reply:
yeah, u are right. i translated just by words.
And i translated it with confuse.
Original translation feels better when i see it now.